examples/deps/WORKSPACE

```diff
@@ -37,24 +34,17 @@ load("@rules_jvm_external//:defs.bzl", "maven_install")
 
 maven_install(
     artifacts = [
-        "androidx.appcompat:appcompat:1.0.0",
-        "junit:junit:4.12",
-        "androidx.test.espresso:espresso-core:3.1.1",
-        "org.hamcrest:hamcrest-library:1.3",
-        "org.jetbrains.kotlinx:kotlinx-serialization-runtime:1.0-M1-1.4.0-rc",
-        "com.google.dagger:dagger:2.45",
-        "com.google.dagger:dagger-compiler:2.45",
-        "com.google.dagger:dagger-producers:2.45",
         "com.google.auto.value:auto-value:1.6.5",
         "com.google.auto.value:auto-value-annotations:1.6.5",
-        "org.robolectric:robolectric:4.7.3",
     ],
     repositories = [
         "https://maven.google.com",
         "https://repo1.maven.org/maven2",
     ],
 )
 
+register_toolchains("//bzl:experimental_toolchain")
```

Comment:
Why does this need to be registered here in the example app? Is this requirement moving forward?

Reply:
Defining a toolchain and not registering it means it will never be chosen.

App didn't compile without the toolchain -- complained about all sorts of things.